### ISSUE TITLE
Local

### ISSUE THREAD

**Issue by nahuelPascual** [guidoferrari/UTN-DDS#2]
* La abstracción `Local` no era necesaria; ojo con sobre-diseñar.
* De nuevo, te faltó el `private`. Atención al encapsulamiento.
* Tratá de evitar los `for` (aunque sea un hermoso `forEach`); usamos Java8 y estaría bueno aprovechar los `stream`s y lambas.
* Ojo con el `System.out.println`; evitalo siempre que puedas... mejor usar algún logger. Para este ejercicio, está perfecto, no hay problema, pero tenelo en cuenta para proyectos reales.

**Comment by guidoferrari** [comment]
- Sinceramente no estaba seguro las pautas específicas, por ende no quise armar la lógica del local en el main. De todos modos lo tomo.
- Again, se me pasó el private. Tengo que prestar más atención.
- Esto del stream S es falta de práctica. Voy a tratar de implementarlo más.
- Tomo el comentario del logger.

Gracias.

**Comment by nahuelPascual** [comment]
Sí, olvidate! Yo también estaba laburando en Java6 cuando la cursé y las lambdas ni las usaba... es práctica nomas

**Comment by nahuelPascual** [comment]
No hace falta que corrijas nada de todo lo que te marqué eh (si querés hacerlo, genial). La idea es solamnete poder charlarlo y plantear los temas,
